src/improvements/tasks/sep/011-opcm-upgrade-prestate-v300-base/VALIDATION.md

```diff
@@ -0,0 +1,163 @@
+# Validation
+
+This document can be used to validate the inputs and result of the execution of the upgrade transaction which you are
+signing.
+
+The steps are:
+
+1. [Validate the Domain and Message Hashes](#expected-domain-and-message-hashes)
+2. [Verifying the transaction input](#understanding-task-calldata)
+3. [Verifying the state changes](#state-changes)
+
+## Expected Domain and Message Hashes
+
+First, we need to validate the domain and message hashes. These values should match both the values on your ledger and
+the values printed to the terminal when you run the task.
+
+> [!CAUTION]
+>
+> Before signing, ensure the below hashes match what is on your ledger.
+>
+> ### Nested Safe 1: `0x6AF0674791925f767060Dd52f7fB20984E8639d8`
+>
+> - Domain Hash: `0x6f25427e79742a1eb82c103e2bf43c85fc59509274ec258ad6ed841c4a0048aa`
+> - Message Hash: `0x4cdac20368824e0bf4725fa51adbab279fb9e378750085a2255937b69e4b7c06`
+>
+> ### Nested Safe 2: `0x646132A1667ca7aD00d36616AFBA1A28116C770A`
+>
+> - Domain Hash: `0x1d3f2566fd7b1bf017258b03d4d4d435d326d9cb051d5b7993d7c65e7ec78d0e`
+> - Message Hash: `0x4cdac20368824e0bf4725fa51adbab279fb9e378750085a2255937b69e4b7c06`
+
+## Understanding Task Calldata
+
+This document provides a detailed analysis of the final calldata executed on-chain for the OPCM upgrade to v2.0.0.
+
+By reconstructing the calldata, we can confirm that the execution precisely implements the approved upgrade plan with no unexpected modifications or side effects.
+
+### Inputs to `opcm.upgrade()`
+
+For each chain being upgrade, the `opcm.upgrade()` function is called with a tuple of three elements:
+
+1. Base Sepolia Testnet:
+   - SystemConfigProxy: [0xf272670eb55e895584501d564AfEB048bEd26194](https://github.com/ethereum-optimism/superchain-registry/blob/2c60e5723c64b5a1b58ab72c5d3816927ff9391a/superchain/configs/sepolia/base.toml#L59)
+   - ProxyAdmin: [0x0389E59Aa0a41E4A413Ae70f0008e76CAA34b1F3](https://github.com/ethereum-optimism/superchain-registry/blob/2c60e5723c64b5a1b58ab72c5d3816927ff9391a/superchain/configs/sepolia/base.toml#L60)
+   - AbsolutePrestate: [0x03682932cec7ce0a3874b19675a6bbc923054a7b321efc7d3835187b172494b6](https://github.com/ethereum-optimism/superchain-registry/blob/712a84f44501322ca61901c2729aa3a56726a602/validation/standard/standard-prestates.toml#L10)
```

Comment:
This prestate is not correct.

Reply:
Why not? 
It matches what I [see here](https://www.notion.so/oplabs/Isthmus-Sepolia-Mainnet-1d2f153ee162800880abe1b47910c071?pvs=4).

Reply:
Ah, my bad I thought the link was incorrect. Resolving this comment.